apps/hash-graph/bin/cli/src/subcommand/server.rs

```diff
@@ -94,207 +98,35 @@ pub struct ServerArgs {
-            )
-            .expect("failed to construct base URL"),
-            version: 1,
-        },
-        "Object".to_owned(),
-        Some("A plain JSON object with no pre-defined structure".to_owned()),
-        "object".to_owned(),
-        HashMap::default(),
-    );
-
-    let empty_list = DataType::new(
-        VersionedUrl {
-            base_url: BaseUrl::new(
-                "https://blockprotocol.org/@blockprotocol/types/data-type/empty-list/".to_owned(),
-            )
-            .expect("failed to construct base URL"),
-            version: 1,
-        },
-        "Empty List".to_owned(),
-        Some("An Empty List".to_owned()),
-        "array".to_owned(),
-        HashMap::from([("const".to_owned(), json!([]))]),
-    );
-
-    // TODO: Revisit once an authentication and authorization setup is in place
-    let root_account_id = AccountId::new(Uuid::nil());
-
-    let mut connection = pool
-        .acquire()
-        .await
-        .change_context(GraphError)
-        .map_err(|err| {
-            tracing::error!(?err, "Failed to acquire database connection");
-            err
-        })?;
-
-    // TODO: When we improve error management we need to match on it here, this will continue
-    //  normally even if the DB is down
-    if connection
-        .insert_account_id(root_account_id)
-        .await
-        .change_context(GraphError)
-        .is_err()
-    {
-        tracing::info!(%root_account_id, "tried to create root account, but id already exists");
-    } else {
-        tracing::info!(%root_account_id, "created root account id");
-    }
 
-    // Seed the primitive data types if they don't already exist
-    for data_type in [text, number, boolean, empty_list, object, null] {
-        let title = data_type.title().to_owned();
-
-        let data_type_metadata =
-            OntologyElementMetadata::External(ExternalOntologyElementMetadata::new(
-                data_type.id().clone().into(),
-                ProvenanceMetadata::new(RecordCreatedById::new(root_account_id)),
-                OffsetDateTime::now_utc(),
-            ));
+    /// Starts a server without connecting to the type fetcher
+    #[clap(long, default_value_t = false, conflicts_with_all = ["type_fetcher_host", "type_fetcher_port"])]
+    pub offline: bool,
+}
 
-        if let Err(error) = connection
-            .create_data_type(data_type, &data_type_metadata)
-            .await
-            .change_context(GraphError)
-        {
-            if error.contains::<VersionedUrlAlreadyExists>() {
-                tracing::info!(%root_account_id, "tried to insert primitive {title} data type, but it already exists");
-            } else {
-                return Err(error.change_context(GraphError));
-            }
-        } else {
-            tracing::info!(%root_account_id, "inserted the primitive {title} data type");
+async fn restore_builtin_types<C: AsClient>(mut store: SnapshotStore<C>) -> Result<(), GraphError> {
```

Comment:
`built_in` I think would be more correct

Reply:
[`bf5ab0c` (#2638)](https://github.com/hashintel/hash/pull/2638/commits/bf5ab0c2acead679ca9b9285f9bf3f8bfb0f2fcc)